src/Api/Utils.purs

```diff
@@ -34,12 +35,12 @@ mkRequest
 mkRequest opts = do
   { baseUrl } <- ask
   response <- liftAff $ request $ defaultRequest baseUrl Nothing opts
-  pure $ hush response.body
+  pure $ hush $ rmap _.body response
```

Comment:
Guess I could use
```PureScript
pure $ map _.body $ hush response
```
here to remove the `Data.Bifunctor` dependency.

Reply:
I don't mind the import.